### PR TITLE
Add support for Comments after Line Continuations new in VB 16.

### PR DESCRIPTION
**Pull request by paul1956**
## Summary

Describe your changes here.
Added Comments after Line Continuation Feature from VB 16, there may be other new feature in VB 16 that I didn't cover.
Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by rpetrusha** [comment]
@KathleenDollard, could you review this PR?

File: docs/visual-basic/getting-started/whats-new.md [hunk 1]
```diff
@@ -15,11 +15,14 @@ This topic lists key feature names for each version of Visual Basic, with detail
 
 ## Current version
 
-Visual Basic 15.8 / Visual Studio 2017 Version 15.8
-For new features, see [Visual Basic 15.8](#visual-basic-158)
+Visual Basic 16 / Visual Studio 2019 Version 16.0
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
Visual Basic 16.0 / Visual Studio 2019 Version 16.0  
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 2]
```diff
@@ -15,11 +15,14 @@ This topic lists key feature names for each version of Visual Basic, with detail
 
 ## Current version
 
-Visual Basic 15.8 / Visual Studio 2017 Version 15.8
-For new features, see [Visual Basic 15.8](#visual-basic-158)
+Visual Basic 16 / Visual Studio 2019 Version 16.0  
+For new features, see [Visual Basic 16](#visual-basic-16)
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
For new features, see [Visual Basic 16.0](#visual-basic-160)
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 3]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
 Visual Basic / Visual Studio .NET 2002
 The first release of Visual Basic .NET
 
+## Visual Basic 16
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
## Visual Basic 16.0
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 4]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
...
+
+In Visual Basic 15. and Earlier comment are only allowed on blank lines before a statement, at the end of a statement or in specific places within a statement where an implicit line continuation is allowed.
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
In Visual Basic 15.8 and earlier versions, comments are only allowed on blank lines, at the end of a statement, or in specific places within a statement where an implicit line continuation is allowed.
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 5]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
...
+
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]: There shouldn't be a paragraph break; this should be a single paragraph.

File: docs/visual-basic/getting-started/whats-new.md [hunk 6]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
...
+Starting is Visual Basic 16 comments are also allowed after Explicit Line Continuation and within a statement on a line beginning with a space followed by an underscore.
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
Starting with Visual Basic 16.0, comments are also allowed after explicit line continuations and within a statement on a line beginning with a space followed by an underscore.
```

File: docs/visual-basic/getting-started/whats-new.md [hunk 7]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
...
+```VB
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion

```vb
```

File: docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md [hunk 8]
```diff
@@ -27,7 +27,7 @@ When writing your code, you might at times create lengthy statements that necess
   
 ### To break a single statement into multiple lines  
   
-- Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return).  
+-   Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return) or starting with language version 16 a comment is allowed after a line continuation..
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]:
```suggestion
-   Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return) or (starting with version 16.0) a comment followed by a carriage return. 
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for updating the documentation, @paul1956. Pending a review from @KathleenDollard, I've left some suggested changes.

File: docs/visual-basic/getting-started/whats-new.md (hunk 3 above)
Comment (KathleenDollard) [reply docs/visual-basic/getting-started/whats-new.md]: Yes, it should be 16.0 in all cases

File: docs/visual-basic/getting-started/whats-new.md [hunk 9]
```diff
@@ -53,6 +56,21 @@ Bit-shift operators, loop variable declaration
 Visual Basic / Visual Studio .NET 2002
 The first release of Visual Basic .NET
 
+## Visual Basic 16
+
```
Comment (KathleenDollard) [inline docs/visual-basic/getting-started/whats-new.md]:
```suggestion
Visual Basic 16.0 focuses on supplying more of the features of the Visual Basic Runtime (microsoft.visualbasic.dll) to .NET Core and is the first version of Visual Basic focused on .NET Core. Many portions of the Visual Basic Runtime depend on WinForms and these will be added in a later version of Visual Basic. 

**Comments allowed in more places within statements**
```

@paul1956 Do you have a better header here?

**Review by KathleenDollard — COMMENTED** [review]
@paul1956 Thanks! 

I like all of Ron's suggestions. 

Paul, can you review the header I added for the line continuation section?

Looks good!

I'll add details of what exactly is added to Core in 16.0 as soon as I can review our final changes and make sure we're done.

File: docs/visual-basic/getting-started/whats-new.md (hunk 9 above)
Comment (paul1956) [reply docs/visual-basic/getting-started/whats-new.md]: With some minor changes many WinForms applications with work, a reasonable set of instructions with necessary changes could be linked to. Also what was listed as 15.8 is in VB16 and should not be lost. The main thing missing is the My namespace and MsgBox listing it acknowledges what issues people might face.

File: docs/visual-basic/getting-started/whats-new.md (hunk 5 above)
Comment (paul1956) [reply docs/visual-basic/getting-started/whats-new.md]: Fixed

**Comment by paul1956** [comment]
The file still refers to 15.8 even though there is no such thing and you can't set language version to 15.8, the feature listed there is part of a minor update and is in VB 16.0 Also there is another new visible feature in VB 16.0 UnconstrainedTypeParameterInConditional and it does not appear anywhere and I don't understand it enough to write about it.

**Comment by rpetrusha** [comment]
@KathleenDollard, we should resolve the issue that @paul1956 is raising about Visual Basic 15.8. Aside from that, the information you've added on Visual Basic 16.0 looks good, @paul1956, so I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**